internal/common/distro.go

```diff
@@ -26,3 +26,16 @@ func VersionLessThan(a, b string) bool {
 
 	return aV.LessThan(bV)
 }
+
+func VersionGreaterThanOrEqual(a, b string) bool {
+	aV, err := version.NewVersion(a)
+	if err != nil {
+		panic(err)
+	}
+	bV, err := version.NewVersion(b)
+	if err != nil {
+		panic(err)
+	}
+
+	return aV.GreaterThanOrEqual(bV)
```

Comment:
Can't we just code this as `return !VersionLessThan(a, b)`? That would save us a few lines of code.

Reply:
We can, I've added a commit that changes it into this suggestion.